ivy/functional/frontends/torch/tensor.py

```diff
@@ -437,6 +437,10 @@ def logical_not_(self):
     def logical_or(self, other):
         return torch_frontend.logical_or(self, other)
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("bfloat16",)}, "torch")
```

Comment:
Seems like the tests are failing due to an unsupported dtype not listed here. You could check the tests and add the unsupported dtype here.

Reply:
Which tests are failing? To me it shows that only run_test (1) is failing which should be ignored as per the instructions above?

Reply:
Not sure what you mean by unsupported dtype not listed, do you mean to remove the line 440? where do you want me to add unsupported dtype?